roles/logs_client/tasks/firewall.yml

```diff
@@ -10,6 +10,19 @@
   register: create_iptables_extras_d
   loop: "{{ logs_class_servers }}"
   become: true
+  ignore_errors: true
+  ignore_unreachable: true
```

Comment:
Why ignoring errors? That defeats the purpose of "ensuring" the path exists...

Reply:
Its correct like this. It ignores some of the nodes being down and fails if all are down.